Description: Ensure that missing optional libraries for the build of x0vncserver are now fatal build errors.
Author: Joachim Falk <joachim.falk@gmx.de>
Forwarded: not-needed

--- a/unix/x0vncserver/CMakeLists.txt
+++ b/unix/x0vncserver/CMakeLists.txt
@@ -31,25 +31,25 @@
 if(X11_XTest_LIB)
   target_link_libraries(x0vncserver ${X11_XTest_LIB})
 else()
-  message(WARNING "No XTest extension.  x0vncserver will be view-only.")
+  message(FATAL_ERROR "No XTest extension.  x0vncserver will be view-only.")
 endif()
 
 if(X11_Xdamage_LIB)
   target_link_libraries(x0vncserver ${X11_Xdamage_LIB})
 else()
-  message(WARNING "No DAMAGE extension.  x0vncserver will have to use the slower polling method.")
+  message(FATAL_ERROR "No DAMAGE extension.  x0vncserver will have to use the slower polling method.")
 endif()
 
 if(X11_Xfixes_LIB)
   target_link_libraries(x0vncserver ${X11_Xfixes_LIB})
 else()
-  message(WARNING "No XFIXES extension.  x0vncserver will not be able to show cursors.")
+  message(FATAL_ERROR "No XFIXES extension.  x0vncserver will not be able to show cursors.")
 endif()
 
 if(X11_Xrandr_LIB)
   target_link_libraries(x0vncserver ${X11_Xrandr_LIB})
 else()
-  message(WARNING "No Xrandr extension.  x0vncserver will not be able to resize session.")
+  message(FATAL_ERROR "No Xrandr extension.  x0vncserver will not be able to resize session.")
 endif()
 
 target_link_libraries(x0vncserver ${X11_LIBRARIES})
